front_office/scroll_down_page/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
delete this app, this functionality is part of base setup and we will not need an app for that. 
Just delete this scroll_down_page folder.

Reply:
will do.